python/private/py_wheel.bzl

```diff
@@ -17,6 +17,62 @@
 load("//python/private:stamp.bzl", "is_stamping_enabled")
 load(":py_package.bzl", "py_package_lib")
 
+RequirementsInfo = provider(
+    "Information about all the requirements of a py_wheel.",
+    fields = {
+        "requirements": "The list of requirements.",
+    },
+)
+
+def _requirements_collector_impl(_, ctx):
+    """Aspect to collect third-party dependencies.
+
+    Args:
+        _: The bazel target the aspect is applied to.
+        ctx: The aspect context.
+    Returns: RequirementsInfo
```

Comment:
nit: I'm surprised stardoc didn't complain about this. Well, only half surprised. Sometimes it demands "\nReturns:\n", and sometimes not, for reasons that aren't clear to me.

Reply:
Addressed in 3295424. This was a carry-over, and it's not necessary. I never add stardoc to rule implementations.